src/prefect_server/api/flows.py

```diff
@@ -601,6 +601,11 @@ async def schedule_flow_runs(flow_id: str, max_runs: int = None) -> List[str]:
             parameters=event.parameter_defaults,
             labels=event.labels,
             idempotency_key=idempotency_key,
+            flow_run_name=event.flow_run_name_template.format(
+                **dict(event.parameter_defaults)
+            )
+            if event.flow_run_name_template is not None
+            else None,
```

Comment:
As written, we will allow `flow_run_name_template` to be `""` generating empty flow run names. I don't think this is handled in Core either. It may be worth just saying

```suggestion
            flow_run_name=event.flow_run_name_template.format(
                **dict(event.parameter_defaults)
            )
            if event.flow_run_name_template else None,
```

Reply:
Good catch @madkinsz - if we do decide to perform string templating, this should be the property of the Clock / Schedule and not the scheduler IMO, which would mean that this logic should move to Core.